client/my-sites/plans-features-main/index.jsx

```diff
@@ -417,15 +410,7 @@ export class PlansFeaturesMain extends Component {
 	}
 
 	getKindOfPlanTypeSelector( props ) {
-		if ( props.isInSignup || props.eligibleForWpcomMonthlyPlans || props.redirectToAddDomainFlow ) {
-			return 'interval';
-		}
-
-		if ( props.withWPPlanTabs && ! props.hidePersonalPlan ) {
-			return 'customer';
-		}
-
-		return 'interval';
+		return props.planTypeSelector;
```

Comment:
I've left this as a function to reduce the amount of change, but we might want to get rid of it, and potentially the 'customer' view altogether. 

For now, it seemed safer to leave it like this in case we wanted to add some more logic to display a different plan layout.